### PR TITLE
Revert "Upgrade Node to 16.15.1"

### PR DESCRIPTION
**Pull request by marijnkampf**
Reverts uktrade/data-hub-frontend#4597
Build error in Jenkins
11:57:41     2022-06-08T10:57:23.65+0000 [STG/0] OUT -----> Nodejs Buildpack version 1.7.70
11:57:41     2022-06-08T10:57:23.81+0000 [STG/0] OUT -----> Installing binaries
11:57:41     2022-06-08T10:57:23.81+0000 [STG/0] OUT engines.node (package.json): 16.15.1
11:57:41     2022-06-08T10:57:23.81+0000 [STG/0] OUT engines.npm (package.json): 8.11.0
11:57:41     2022-06-08T10:57:23.81+0000 [STG/0] OUT **ERROR** Unable to install node: no match found for 16.15.1 in [12.22.10 12.22.12 14.19.1 14.19.2 16.14.2 16.15.0]
11:57:41     2022-06-08T10:57:23.82+0000 [STG/0] ERR Failed to compile droplet: Failed to run all supply scripts: exit status 14
11:57:41     2022-06-08T10:57:23.82+0000 [STG/0] OUT Exit status 223
11:57:41     2022-06-08T10:57:24.30+0000 [STG/0] OUT Cell e0ed574c-b63a-467e-8183-cbb8c01b4100 stopping instance c85b91bb-2204-4ec5-aadd-e8e8938f2064
11:57:41     2022-06-08T10:57:24.30+0000 [STG/0] OUT Cell e0ed574c-b63a-467e-8183-cbb8c01b4100 destroying container for instance c85b91bb-2204-4ec5-aadd-e8e8938f2064
11:57:41     2022-06-08T10:57:24.35+0000 [API/5] ERR Failed to stage build: staging failed

### CODE REVIEW

**Comment by marijnkampf** [comment]
I presume that this is a similar issue as with Docker in that the latest version 16.15.1 isn't available for Jenkins yet.

**Review by santoshdasa12345 — APPROVED** [review]

File: test/functional/cypress/specs/companies/investments/investment-large-capital-investor-details-spec.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ describe('View large capital investor details page', () => {
         required_checks_conducted: '9beab8fc-1094-49b4-97d0-37bc7a9de631',
         required_checks_conducted_on: `${lastMonth.getFullYear()}-${(
           lastMonth.getMonth() + 1
-        ).toString()}-${lastMonth.getDate().toString()}`,
+        ).toString()}-${lastMonth.getDay().toString()}`,
```
Comment (craigjbass) [inline test/functional/cypress/specs/companies/investments/investment-large-capital-investor-details-spec.js:172]: Does this return a different value in node 16.15.1?

**Review by craigjbass — CHANGES_REQUESTED** [review]
Looks good, although just double check you want to change the tests back

Secondly you're changing the Dockerfile.dependencies back but it doesn't look like a release was made that would have actually applied this, so is it broken on master in local dev currently?

**Review by craigjbass — APPROVED** [review]
accidentally hit request changes

**Pull request merged**